common/gpu/amd/default.nix

```diff
@@ -1,24 +1,13 @@
-{ config, lib, ... }:
+{ lib, ... }:
 
 {
-  options.hardware.amdgpu.loadInInitrd = lib.mkEnableOption (lib.mdDoc
```

Comment:
We also need to fix this in all our modules.

Reply:
I will get that fixed

Reply:
Fixed this and reenabled it by default when the gpu/amd/default is imported.